packages/sw-cli/src/lib/templates/sw.js.tmpl

```diff
@@ -0,0 +1,6 @@
+importScripts(
+  "<%= manifestPath %>",
```

Comment:
Yeah I haven't tackled this problem yet.

I'm still in favor of adding a timestamp in the top of the service worker, this is generic and safe and results in a new service worker.

Secondly, if we follow this approach, using the CLI to rebuild the manifest can then update the service worker timestamp, throwing an error (or simply not updating the manifest file) if there is any issue in doing that (i.e. sw file can't be found, no existing timestamp is found or .....).

If we inline the contents we can ONLY rewrite the service worker file which prevents people from being able to edit the service worker directly which is :-1: for me.

What do you think?